image/generic/sle16/config.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+#================
+# FILE          : config.sh
+#----------------
+# PROJECT       : KIWI - Image System
+# COPYRIGHT     : (c) 2023 SUSE Software Solutions Germany GmbH
+#               :
+# AUTHORS       : Marcus Schaefer <ms@suse.de>
+#               : Keith Berger <kberger@suse.com>
+#               :
+# BELONGS TO    : Operating System images
+#               :
+# DESCRIPTION   : configuration script for SUSE based
+#               : operating systems
+#----------------
+#======================================
+# Functions...
+#--------------------------------------
+test -f /.kconfig && . /.kconfig
+test -f /.profile && . /.profile
```

Comment:
Please drop the reading of .profile. We do that in kiwi automatically for every script call

Also please add

```
set -eux
```

At the beginning of the script code, we want to fail on errors and not silently ignore them